### PR TITLE
feat: python multipart

### PR DESCRIPTION
**Pull request by loks0n**

### CODE REVIEW

File: templates/python/package/payload.py.twig [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+                    lines = lines[1:]  # Remove the Content-Type line
+                    headers = json.loads(''.join(lines))
+                    data[name] = {header['name']: header['value'] for header in headers}
+                else:
+                    data[name] = ''.join(lines) if lines else ''
+
+        data['responseStatusCode'] = int(data.get('responseStatusCode', 0))
+        data['duration'] = float(data.get('duration', 0))
+        data['responseBody'] = cls.from_string(data.get('responseBody', ''))
+        return data
```
Comment (christyjacob4) [inline templates/python/package/payload.py.twig:36]: @loks0n are people able to then do `response.responseBody.toJSON()` or `response.responseBody.toBinary()` for example ?

File: templates/python/package/payload.py.twig [hunk 2]
```diff
@@ -0,0 +1,96 @@
+import os
```
Comment (christyjacob4) [inline templates/python/package/payload.py.twig:1]: @loks0n I'm pretty sure we need changes in the service.py especially in the functions for createFile, createDeployment and createExecution to use the new payload class. 

Please test these and let me know.

File: templates/python/package/payload.py.twig [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    @classmethod
+    def from_path(cls, path):
+        instance = cls()
+        instance.path = path
+        instance.filename = os.path.basename(path)
+        instance.mime_type = mimetypes.guess_type(path)[0]
+        return instance
+
+    @classmethod
+    def from_bytes(cls, bytes, filename, mime_type=None):
```
Comment (christyjacob4) [inline templates/python/package/payload.py.twig:47]:
```suggestion
    def from_binary(cls, bytes, filename, mime_type=None):
```

File: templates/python/package/payload.py.twig (hunk 1 above)
Comment (loks0n) [reply templates/python/package/payload.py.twig:36]: `to_json()` and `to_binary()` but yeah

File: tests/languages/ruby/plan.ts
Comment (christyjacob4) [inline tests/languages/ruby/plan.ts:1]: I think we can remove this right?

File: templates/python/package/payload.py.twig [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+        self._data = data
+        self.filename = data.filename
+
+    @classmethod
+    def from_path(cls, path: str, filename: Optional[str] = None):
+        return cls(FileData(path, filename=filename))
+
+    @classmethod
+    def from_binary(cls, b: bytes, filename: Optional[str] = None):
+        return cls(MemoryData(b, filename=filename))
```
Comment (christyjacob4) [inline templates/python/package/payload.py.twig:53]: these method also need to take an optional mime type. Please check the PHP PR https://github.com/appwrite/sdk-generator/pull/940

File: tests/languages/ruby/plan.ts
Comment (christyjacob4) [inline tests/languages/ruby/plan.ts:1]: @loks0n I think we can remove this?

**Review by christyjacob4 — APPROVED** [review]

File: tests/languages/ruby/plan.ts
Comment (loks0n) [reply tests/languages/ruby/plan.ts:1]: done

**Pull request merged**